spec/book_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../book'
+
+describe Book do
+  context 'Unit testing in the Book class' do
+    it 'test creating an instance' do
+      book = Book.new('test_title', 'test_author')
+      expect(book.title).to eq 'test_title'
+      expect(book.author).to eq 'test_author'
+      expect(book.rentals).to eq []
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Great job so far team 👍🏽 . Kindly add a test for the `add_rental` method as stated by the project requirements.